packages/terra-clinical-onset-picker/tests/nightwatch/OnsetPickerTestRoutes.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Route } from 'react-router';
+import OnsetPickerTests from './OnsetPickerTests';
+
+import DefaultOnset from './component/Default';
+import LessMonth from './component/LessMonth';
+import LessYear from './component/LessYear';
+
+const routes = (
```

Comment:
I feel like we'd want to add tests for Unknown, Age, Month, Year, Date selections since the rendered dropdown and input change.

Additionally tests for onChange callbacks, maxCount calculation, etc to ensure they are working as expected.

Reply:
I do already have tests about the Unknown precision and Age, Month, Year, and Date granularities. I will be supplying onChange and util tests shortly.